pysmartdatamodels/setup.py

```diff
@@ -14,7 +14,13 @@
 LONG_DESCRIPTION = (HERE / "README.md").read_text()
 LONG_DESC_TYPE = "text/markdown"
 
-INSTALL_REQUIRES = ["collections", "json", "jsonref", "jsonschema", "pytz", "requests", "sys", "validate", "datetime"]
+INSTALL_REQUIRES = [
+    # "collections", "json" # These are builtins
+    "jsonref", "jsonschema", "pytz", "requests", 
+    # "sys", # another builtin
+    # "validate", # Does not appear to be used
```

Comment:
datetime is used:
https://github.com/smart-data-models/data-models/blob/master/pysmartdatamodels/pysmartdatamodels.py#L349